src/pages/index.tsx

```diff
@@ -116,7 +117,13 @@ export function Index(props: RouteComponentProps) {
                     </Box>
                   ))}
                 </Box>
-                <Text sx={{ lineHeight: '1' }}>{palette.name}</Text>
+                <Text sx={{
+                  lineHeight: '1',
+                  textOverflow: 'ellipsis',
+                  width: '80%',
```

Comment:
Doesn't a percentage width here mean there will be a variable amount of space between the text and the button?

Reply:
Since the grid items don't have fixed width (`gridTemplateColumns: 'repeat(auto-fill, minmax(300px, 1fr))',`), the text also does not have a fixed width, I just put 80% to be on the safe side. Please correct me if I'm wrong.

Reply:
I think I missed your point, I don't know if it's a bad practice to have the spacing between the text and button change. Here I put 80% to avoid the button covering the text. Wdyt?